docs/releases/5.1.md

```diff
@@ -535,3 +535,12 @@ Additionally, two new events will be dispatched when the dialog visibility chang
 | ------ | ----------------- |
 | Show   | `w-dialog:shown`  |
 | Hide   | `w-dialog:hidden` |
+
+## Shared template `.../tables/attrs.html` has been renamed to `.../shared/attrs.html`
```

Comment:
Sorry, missed this during review – this should've been heading level 3

Reply:
I'll fix and push to master. Sorry I missed that one also.

Reply:
Fixed aa52f84ec47faa9abf94c11207a7e459fbb3201b